### PR TITLE
fix: Switch to use kong/httpbin to support arm64. Fixes #10427

### LINKED ISSUE #10427: Replace `httpbin` with something up to date
`httpbin` is no longer maintained and does not support arm64 architecture.

image

---

**Message from the maintainers**:

Love this enhancement proposal? Give it a 👍. We prioritise the proposals with the most 👍.

### PR DESCRIPTION
**Pull request by sonbui00**
Fixes #10427

### Motivation
Run local development with httpbin on Mac M1 (ard64)

### Modifications
Replace out date image httpbin with ~~mccutchen/go-httpbin~~ kong/httpbin

### Verification
Tested locally/manually + Run `make test`

### CODE REVIEW

File: manifests/quick-start/base/httpbin/httpbin-deploy.yaml [hunk 1]
```diff
@@ -16,7 +16,12 @@ spec:
       automountServiceAccountToken: false
       containers:
         - name: main
-          image: kennethreitz/httpbin
+          image: mccutchen/go-httpbin
```
Comment (terrytangyuan) [inline manifests/quick-start/base/httpbin/httpbin-deploy.yaml:19]: Are there multiple alternatives? If so, what's the reason to pick this one?

Comment (sonbui00) [reply manifests/quick-start/base/httpbin/httpbin-deploy.yaml:19]: I saw this one has good maintenance. But the test is failing due to a different response on oaurth2. I'm considering finding other alternatives or rebuilding this docker image.

Comment (sonbui00) [reply manifests/quick-start/base/httpbin/httpbin-deploy.yaml]: @terrytangyuan help me review again. Use a new image kong/httpbin that support multiple arch
I checked 2 failing test, they do not relate with my change

Comment (sonbui00) [reply manifests/quick-start/base/httpbin/httpbin-deploy.yaml]: @terrytangyuan just run test on local for (2) and passed that test. nodejs v19 link for (1) also available now. Plese help me retrigger github action for test again

**Review by terrytangyuan — APPROVED** [review]

**Pull request merged**